uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -27,22 +35,55 @@ public class MqttControlSteps {
 
     private static final String DEFAULT_CLIENT_DEVICE_POLICY_CONFIG = "/configs/iot/basic_client_device_policy.yaml";
 
+    private static final int DEFAULT_CONTROL_GRPC_PORT = 47_619;
```

Comment:
It is possible to use autoselected port and pass it to client's config ? 
At least in future.

Reply:
It is possible